packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -133,6 +133,8 @@ public void shutdown() {
   private @Nullable ViewGroupDrawingOrderHelper mDrawingOrderHelper;
   private float mBackfaceOpacity;
   private String mBackfaceVisibility;
+  private boolean mIsTransitioning = false;
+  private @Nullable Set<Integer> mChildrenRemovedWhileTransitioning = null;
```

Comment:
I'm using `Set<Integer>` (holding only the view tag) instead of `Set<View>` to avoid any retain issues.

Reply:
Consider using androidx.collection.MutableIntSet instead to avoid Integer boxing.

Reply:
Thanks, I was not aware of this API! Hover it seems that it has been added few months ago in [`androidx.collection.MutableIntSet` 1.4.0](https://developer.android.com/reference/androidx/collection/MutableIntSet), while my development project (using recent react-native 0.76) uses version 1.1.0. I'm not sure where would I need to bump / add this dependency in RN to make use of this.